src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -89,10 +89,8 @@
                 INTAKE_SOLENOID_DOWN = 0;
 
         // index can ids are range 40-49
-        public static final int INDEX_INGEST_MOTOR = 40, INDEX_FEEDER_MOTOR = 41, INGEST_RED = 0, INGEST_BLUE = 1,
-                INGEST_PROXIMITY = 2, FEEDER_RED = 3, FEEDER_BLUE = 4, FEEDER_PROXIMITY = 5, INGEST_TOP_RED = 6,
-                INGEST_TOP_BLUE = 7,
-                INGEST_TOP_PROXIMITY = 2;
+        public static final int INDEX_INGEST_MOTOR = 40, INDEX_FEEDER_MOTOR = 41, INGEST_PROXIMITY = 8,
+                FEEDER_PROXIMITY = 9;
```

Comment:
I mentioned this is #132, but maybe we should move `INGEST_PROXIMITY` to the index section and rename it and `FEEDER_PROXIMITY` accordingly.

Reply:
I assume you meant move `INGEST_PROXIMITY` to the intake section?